linux/step05_centos7_nginx.sh

```diff
@@ -1,15 +1,19 @@
 #!/bin/bash
 
+#start-copy
+cp setup_omero_nginx.sh ~omero
+#end-copy
+
 #start-install
 yum -y --enablerepo=cr install nginx
```

Comment:
for C6, yes the version is old.

We recommend version in our requirements page for java, ice, pg etc. in the doc.
The version installed in the installation script should match what is recommended, for nginx we recommend 1.8/1.9 so that what we should/install test.
That's another reason why we are also thinking of moving the sysadmin doc under omero-install so we will hopefully keep things in synch (it has not been the case!)

note that I am currently fixing the ubuntu install script since I just noticed that 1.4.6 is currently installed.